lib/ansible/modules/system/user.py

```diff
@@ -48,6 +48,13 @@
         required: false
         description:
             - Optionally sets the I(UID) of the user.
+    hidden:
+        required: false
+        default: 0
+        choices: [ 0, 1 ]
+        description:
+            - Optionally hide the user from the login window and system
+              preferences on macOS.
```

Comment:
We should make this a bool and default to null.  Then we can tell whether the user is asking to change it or not.  (True and False are the user requesting to set the state.  None means leave it alone).

I'd reformat the description slightly:

> Hide the user from the login window.  This is currently only implemented on MacOSX